### PR TITLE
Correct timing constraint failure check

### LINKED ISSUE #168: Vivado log parsing produces wrong timing result with large negative slack
The `ParseTimingSummary` script does not detect that timing was not met in this particular instance.

```
 WNS(ns)      TNS(ns)  TNS Failing Endpoints  TNS Total Endpoints      WHS(ns)      THS(ns)  THS Failing Endpoints  THS Total Endpoints     WPWS(ns)     TPWS(ns)  TPWS Failing Endpoints  TPWS Total Endpoints  
    -------      -------  ---------------------  -------------------      -------      -------  ---------------------  -------------------     --------     --------  ----------------------  --------------------  
     -2.408    -1161.238                    963                 4943        0.038        0.000                      0                 4943        1.520        0.000                       0                  3932  
```

### PR DESCRIPTION
**Pull request by martijnbastiaan**
Fixes #168

### CODE REVIEW

File: bittide-instances/bin/Shake.hs [hunk 1]
```diff
@@ -166,11 +166,10 @@ main = do
...
           -- after.
-          report <- liftIO (ParseTimingSummary.parseFile postRouteTimingSummaryPath)
-          case ParseTimingSummary.meetsTiming report of
-            Nothing -> pure ()
-            Just wns -> error [I.i|
-              Design did not meet timing. Negative WNS detected: #{wns}. Check out
+          liftIO $ unlessM
+            (meetsTiming postRouteTimingSummaryPath)
+            (error [I.i|
+              Design did not meet timing. Negative WNS detected. Check out
```
Comment (lmbollen) [inline bittide-instances/bin/Shake.hs:172]: Since you don't do any WNS detection anymore, I think you should rephrase this to just say that timing constraints were not met.

**Review by lmbollen — APPROVED** [review]

File: bittide-instances/bin/Shake.hs (hunk 1 above)
Comment (martijnbastiaan) [reply bittide-instances/bin/Shake.hs]: Done!
